src/QmlControls/CenterPoint.qml

```diff
@@ -0,0 +1,182 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.14
+import QtPositioning 5.14
+
+Rectangle {
+    id: root
+    color: '#000000'   // black
+
+    property var rootWidth: null;
+    property var rootHeight: null;
+
+    width: rootWidth / 3.24
+    height: rootHeight / 1
+    Text {
+        id: centerPointText
+        color: "#FFFFFF"    // white
```

Comment:
define color property in root item for this color

Reply:
done